pkg/healthcheck/healthcheck_addons.go

```diff
@@ -0,0 +1,76 @@
+package healthcheck
```

Comment:
I like how this was added into a separate file, to avoid bloating more `healthcheck.go` :+1: